src/db/index.ts

```diff
@@ -79,10 +80,10 @@ export const gracefulExit = async (exitCode: number = 0): Promise<void> => {
   })
 }
 
-export const defaultPostConnect = async (): Promise<void> => {
+export const defaultPostConnect = async (): Promise<ChangeStream> => {
   console.log('Kudos!')
   await createIndexes()
-  await streamListener()
+  return await streamListener()
```

Comment:
Hope this is ok. Let me know if there's a better design pattern to use to record the stream so we can close it later.